src/Microsoft.Win32.Primitives/tests/Win32Exception.cs

```diff
@@ -80,7 +80,7 @@ public static void InstantiateException()
         private const int E_FAIL = unchecked((int)0x80004005);
 
         [Fact]
-        [PlatformSpecific(PlatformID.Windows)]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Not for this change, but should we come up with a different name for PlatformID in our xunit extensions?  Otherwise eventually we're going to need to disambiguate with the namespace on all uses, once all tests are enabled for .NET Framework.